ts/components/ui/cards/payment/PaymentCardSmall.tsx

```diff
@@ -0,0 +1,201 @@
+import * as React from "react";
+import { Pressable, StyleSheet, View } from "react-native";
+import Animated from "react-native-reanimated";
+import Placeholder from "rn-placeholder";
+import { WithTestID } from "../../../../types/WithTestID";
+import { Icon } from "../../../core/icons";
+import { IOLogoPaymentType } from "../../../core/logos";
+import { VSpacer } from "../../../core/spacer/Spacer";
+import { NewH6 } from "../../../core/typography/NewH6";
+import { IOColors } from "../../../core/variables/IOColors";
+import { IOBannerRadius } from "../../../core/variables/IOShapes";
+import { LogoPaymentOrDefaultIcon } from "../../utils/baseComponents/LogoPaymentOrDefaultIcon";
+import { useInteractiveSpringAnimation } from "../../utils/hooks/useBaseSpringAnimation";
+
+const getRenderData = (
+  props: CardDataType
+): {
+  iconName: IOLogoPaymentType | undefined;
+  bottomText: string;
+} => {
```

Comment:
We should define a type, like `RenderData` for example. What do you think?

```suggestion

type RenderData = {
  iconName: IOLogoPaymentType | undefined;
  bottomText: string;
}

const getRenderData = (
  props: CardDataType
): RenderData => {
```

Reply:
addressed in [aeb2527](https://github.com/pagopa/io-app/pull/4759/commits/aeb2527c92241119b9053c2ba1572f01acef402b)